pom.xml

```diff
@@ -567,6 +567,10 @@
                         <groupId>org.eclipse.jetty.orbit</groupId>
                         <artifactId>javax.servlet</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>com.thoughtworks.paranamer</groupId>
```

Comment:
Could you please explain why did you add this exclusion?

Reply:
Gremlin-spark has a depencency conflict with jackson in both usage locations.